src/components/App.js

```diff
@@ -0,0 +1,11 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+const App = () => (
```

Comment:
* [ ] `App.js` has children components wrapped in React.Fragment short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))
The React convention can make you create nodes that you don't really need sometimes.  With the aid of React fragments no extra node is created, and the code efficiency is improved.
> We are not adding styling yet so we can better appreciate the concept